turbo.json

```diff
@@ -6,17 +6,17 @@
   "tasks": {
     "ready": {
       "dependsOn": ["^ready"],
-      "outputs": ["dist/**", "build/**"]
+      "outputs": ["../../dist/**", "dist/**"]
     },
     "dev": {
       "dependsOn": ["ready"],
-      "outputs": ["dist/**", "build/**", "i18n/locales/**"],
```

Comment:
This breaks the ability to reflect on I18n runtime changes.

Reply:
Can you describe it a little more?

Reply:
Change I18n runtime doesn't work without previous output settings

Reply:
It doesn't change sadly

It still doesn't refreshing plugin, but file in `dist` is changing